packages/jsapi-components/src/spectrum/Picker.tsx

```diff
@@ -1,10 +1,13 @@
 import { PickerNormalized } from '@deephaven/components';
 import { type PickerProps } from './PickerProps';
 import { usePickerProps } from './utils';
+import useTableClose from '../useTableClose';
 
 export function Picker(props: PickerProps): JSX.Element {
   const pickerProps = usePickerProps<PickerProps>(props);
 
+  useTableClose(props.table);
```

Comment:
I think this is the same issue. The parent component using Picker may not want you to close the table. It's passed in as a reference that should be managed elsewhere.